public/studies-schema.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
What's the purpose of this file?

Reply:
We're going to use it on the remote-settings server, but in the near future we can use it in tests and maybe at runtime as well.